gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcWriteChannelTest.java

```diff
@@ -117,7 +119,11 @@ public void setUp() throws Exception {
         .build()
         .start();
     stub =
-        StorageGrpc.newStub(InProcessChannelBuilder.forName(serverName).directExecutor().build());
+        StorageGrpc.newStub(
+            InProcessChannelBuilder.forName(serverName)
+                .intercept(new InvocationIdGrpcChannelInterceptor())
```

Comment:
why can't we use MetadataUtils.newAttachHeadersInterceptor?

Reply:
explained above.